lib/core/di/di_setup.dart

```diff
@@ -84,6 +84,10 @@ Future<void> setupDependencies() async {
   // Firebase Services (Storage, Auth, etc)
   await _registerFirebaseModule();
 
```

Comment:
Note - there is probably a better way to wait for the initialization than this?!

The object can be lazy but we need to wait for init otherwise it's not ready...